client/my-sites/plan-features/index.jsx

```diff
@@ -412,10 +418,26 @@ class PlanFeatures extends Component {
 				features,
 				planName
 			} = properties;
+			
+			let planFeatures = features;
```

Comment:
I used `planFeatures` because `features` was declared in the deconstructing assignment with the `const`. So, the variable itself is inmutable. Then I used `let` to allow mutation of `planFeatures`.

But, now that i think of it, I can just deconstruct the assignment using `let` instead of `const` and just use `features`... I will refactor accordingly, thanks for the input! :)